frontend/src/pages/device/index.vue

```diff
@@ -321,6 +321,9 @@ export default {
                     try {
                         // * Enable Device Editor (Step 1) - (browser->frontendApi) User clicks button to "Enable Editor"
                         const result = await deviceApi.enableEditorTunnel(this.device.id)
+                        if (result.affinity) {
+                            document.cookie = `INGRESSCOOKIE=${result.affinity}`
```

Comment:
As you suggested, lets pick a different cookie name here to avoid tying out hands too much.

How about `FFSESSION` ?